libs/utils/utils.js

```diff
@@ -150,8 +150,11 @@ const PROMO_PARAM = 'promo';
 function getEnv(conf) {
   const { host } = window.location;
   const query = PAGE_URL.searchParams.get('env');
-
   if (query) return { ...ENVS[query], consumer: conf[query] };
+
+  const { clientEnv } = conf;
```

Comment:
I wonder if we should more generically look for `window.miloClientEnv` ? 
That would work for blocks exported outside of the gnav as well

Reply:
Yes, I  will update it to `window.miloClientEnv`

Reply:
@narcis-radu Initially it was added as part of config. Should we keep it this way?
cc: @salonijain3